modules/aruco/misc/pattern_generator/README.md

```diff
@@ -0,0 +1,40 @@
+# OpenCVMarkerPrinter
+
+## Description
+This small app can save some commonly used opencv markers such as ArUco, ArUcoGrid, Chessboard and ChArUco to vector graphics file. **Supported vector graphics file format: .svg, .pdf and .ps. Supported image file format: .png (not recommend).**
```

Comment:
Maybe it is better to remove _png_ support instead of describing why it is bad?

Reply:
`.png` is fine. At least only this format from the whole list can be opened by OpenCV

Reply:
I think remove .png is fine, opencv can draw .png marker it-self, although the real size need to be take care by user itself